shape_optimization/11_Shape_Update_Optimization_Stanford_Bunny/README.md

```diff
@@ -0,0 +1,43 @@
+# Shape Optimization
+Shape update of a wrapping surface to generate a smooth representation of a complex object, which in this example is a Stanford bunny.
```

Comment:
For this example please describe only the small sphere example, as this is the one with better results and it is also the one that is in the top level of the folder.

please crop the gif (https://ezgif.com/crop) and figures such that only that example is shown. 

At the end of the page we can make a small reference to the shrink folder by only showing the gif of the shrinking and stating tat with slight modifications of the optimization setup a shrinking can be achieved.

Reply:
shrink is with Large sphere & growth would be the Small sphere ? right ?

Reply:
yes